tests/chainer_tests/optimizer_hooks_tests/utils.py

```diff
@@ -0,0 +1,39 @@
+import numpy
```

Comment:
Shouldnt this file be in `chainer/testing/` instead of tests

Reply:
Hmm, I don't this this feature deserve a public API.